src/index.html

```diff
@@ -22,6 +22,433 @@
+          alt="logo"
+          class="container__logo"
+        />
+      </a>
+      <nav>
+        <ul class="container__nav-list">
+          <li class="container__link">
+            <a
+              href="#apple"
+              class="is-active container__nav-link"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="container__link">
+            <a
+              href="#samsung"
+              class="container__nav-link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="container__link">
+            <a
+              href="#smartphones"
+              class="container__nav-link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="container__link">
+            <a
+              href="#laptops_computers"
+              class="container__nav-link nav_hover"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="container__link">
+            <a
+              href="#gadgets"
+              class="container__nav-link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="container__link">
+            <a
+              href="#tablets"
+              class="container__nav-link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="container__link">
+            <a
+              href="#photo"
+              class="container__nav-link"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="container__link">
+            <a
+              href="#video"
+              class="container__nav-link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="catalog">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          class="card__imac-image"
+          src="./images/imac.jpeg"
+          alt="imac-image"
+        />
+
+        <h2
+          href="#"
+          class="card__title"
+        >
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__ratings">
+          <div class="stars stars--4">
+            <div class="stars-star"></div>
+            <div class="stars-star"></div>
+            <div class="stars-star"></div>
+            <div class="stars-star"></div>
+            <div class="stars-star"></div>
+          </div>
+
+          <p class="card__reviews">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price-word">Price:</p>
+          <p class="card__price-number">$2,199</p>
+        </div>
+
+        <a
+          class="card__button"
+          data-qa="card-hover"
+        >
+          buy
+        </a>
+      </div>
+      <div
```

Comment:
Add the blank line between the cards
```suggestion
      </div>
      
      <div
```